protocol/directory.go

```diff
@@ -39,18 +65,43 @@ func (d *ConiksDirectory) Update() {
 	}
 }
 
+// SetPolicies sets this ConiksDirectory's epoch deadline and VRF
+// private key.
 func (d *ConiksDirectory) SetPolicies(epDeadline merkletree.TimeStamp, vrfKey vrf.PrivateKey) {
 	d.policies = merkletree.NewPolicies(epDeadline, vrfKey)
 }
 
+// EpochDeadline returns this ConiksDirectory's current epoch deadline
+// as a timestamp.
 func (d *ConiksDirectory) EpochDeadline() merkletree.TimeStamp {
 	return d.pad.LatestSTR().Policies.EpochDeadline
 }
 
+// LatestSTR Returns this ConiksDirectory's latest STR.
```

Comment:
> I think "latest" clearly indicates that we actually perform all operations on the tree which has been signed

This makes sense to me. My argument for "current" is that I think captures the user's intent better, since we don't expect user's to really have a notion of the underlying PAD. But if the three of you feel stronger about using "latest", let's go with that.